boards/common/arduino-atmega/include/periph_conf.h

```diff
@@ -61,6 +63,17 @@ extern "C" {
 #define TIMER_0_ISRB        TIMER1_COMPB_vect
 #endif
 
+#ifdef CPU_ATMEGA32U4
+#define TIMER_NUMOF         (2U)
+
+#define TIMER_0             MEGA_TIMER1
+#define TIMER_0_MASK        &TIMSK1
+#define TIMER_0_FLAG        &TIFR1
+#define TIMER_0_ISRA        TIMER1_COMPA_vect
+#define TIMER_0_ISRB        TIMER1_COMPB_vect
```

Comment:
I'm not sure these are the right config parameters. `tests/periph_timer` fails to initialize `TIMER_0`.

Reply:
Indeed, the configuration of the timers was not good, it is corrected.